winterfell/models/form.js

```diff
@@ -8,7 +8,9 @@ var formSchema = new Schema({
   key: String, // Code name for the form
   responses: Schema.Types.Mixed,
   user: Schema.Types.ObjectId,
-  claim: Schema.Types.ObjectId
+  claim: Schema.Types.ObjectId,
+  answered: Number,
+  answerable: Number
```

Comment:
I think it makes forms a bit more complicated than necessary to divide things into "required" and "optional",  my thought for vetafi is there there will never be any optional questions, which makes it easier for the user, because if you see it, you need to answer it. 

The methods for addressing pieces of information that may or may not be necessary then becomes asking gating questions, and having the "optional" questions become visible depending on the answer to those gating questions. For instance we can ask "Do you have a VA file number?", and then if you answer yes, the question "What is your VA file number?" becomes visible. 

For things like address line 2, we can just have the line being blank be considered "complete" in validation.

What do you think?